client/src/Features/MicrosoftLearn/MicrosoftLearnList.tsx

```diff
@@ -57,42 +57,50 @@ const MicrosoftLearnListInner = ({ styles }: IStylesOnly<MicrosoftLearnListStyle
 
   return useObserver(() => {
     const isLoadingCatalog = !!learnStore.isLoadingCatalog;
-    const catalogContent = learnStore.filteredCatalogContent;
-    const noVisibleItems = !isLoadingCatalog && catalogContent?.length === 0;
+    const { filteredCatalogContent } = learnStore;
+    const noVisibleItems = !isLoadingCatalog && filteredCatalogContent?.length === 0;
 
     return (
       <div className={classes.root}>
         {noVisibleItems ? (
           <Text className={classes.noResultsText}>No items match your search. Please refine your search criteria.</Text>
         ) : (
-          <AutoSizer>
-            {({ height, width }): JSX.Element | null => {
-              if (autoSizerWidth === 0 || Math.abs(autoSizerWidth - width) > 25) {
-                setAutoSizerWidth(width);
-              }
 
-              const numItemsPerRow = Math.floor(autoSizerWidth / FIXED_ITEM_WIDTH);
-              const rowCount = catalogContent
-                ? Math.floor(catalogContent.length / numItemsPerRow) + (catalogContent.length % numItemsPerRow ? 1 : 0)
-                : 2;
-              return (
-                <FixedSizeList
-                  style={getListStyle(isLoadingCatalog)}
-                  height={height}
-                  itemCount={rowCount}
-                  itemData={{
-                    numItemsPerRow,
-                    itemsData: catalogContent,
-                    isLoadingCatalog
-                  }}
-                  itemSize={FIXED_ITEM_HEIGHT}
-                  width={autoSizerWidth}
-                >
-                  {ListRow}
-                </FixedSizeList>
-              );
-            }}
-          </AutoSizer>
+          <>
+            {filteredCatalogContent && filteredCatalogContent.length>0 &&
+              (<div className={classes.contentCount}>
+              <Text variant="mediumPlus" className='contentCountText'> {filteredCatalogContent.length.toLocaleString()} results from Microsoft Learn </Text>         

```

Comment:
just curious to know why did we need to use `toLocaleString` here?

Reply:
It formats the number of course contents (of type `Number`) to a `string` with commas at appropriate places.